app/views/data_portal/index.html.erb

```diff
@@ -37,7 +37,9 @@
       <section class="l-dataset -finscope" id="finscope-dataset">
         <div class="row data-portal-row">
           <div class="grid-s-12 grid-m-6">
-            <h3 class="title -big -dark">National Surveys</h3>
+            <a href=".l-country-list" class="js-anchor">
+              <h3 class="title -big -dark" onclick="showFinscope()">National Surveys</h3>
```

Comment:
We tend to avoid online js for organization purposes. 
This logic, along with the functions you created at the end of the file should live in a js view. 

I think this file: `scripts/pages/data_portal/DataPortalIndexPage.js`might be the one that should keep this logic.